src/main/java/frc/robot/Robot.java

```diff
@@ -177,6 +177,8 @@ public void teleopPeriodic() {
     driveTrain.arcadeDrive(
       -controlInputs.driveStickY*forwardPower,
       controlInputs.driveStickX*turnPower);
+
+    SmartDashboard.putNumber("Motor Power", (-controlInputs.driveStickY*forwardPower));
```

Comment:
Don't forget we are squaring the inputs when we call the WPI lib class to set the motor power, so this isn't 100% accurate.